dandi/models.py

```diff
@@ -396,9 +396,29 @@ class BioSample(DandiBaseModel):
     )
     anatomy: Optional[List[Anatomy]] = Field(
         None,
-        description="UBERON based identifier for the location of the sample",
+        description="UBERON based identifier for what organ the sample belongs "
+        "to. Use the most specific descriptor.",
         nskey="dandi",
     )
+
+    wasDerivedFrom: Optional[List["BioSample"]] = Field(None, nskey="prov")
```

Comment:
this recursive formulation should allow us to have cell ids, slice ids, and tissue ids.